apps/docs/app/features/content-menu/ContentMenu.tsx

```diff
@@ -43,7 +43,7 @@ export const ContentMenu = forwardRef((_, ref) => {
         const subItems = item.subItems?.filter((subItem) => subItem.url);
         const hasSubItems = Boolean(subItems?.length);
         return (
-          <AccordionItem key={item.title}>
+          <AccordionItem key={item.title} mt={0.5} mb={0.5} >
```

Comment:
Her kan du bruke `my` for å spare deg noen tegn.
```suggestion
          <AccordionItem key={item.title} my={0.5}>
```

Reply:
Fikse